src/common/messaging.ts

```diff
@@ -38,6 +38,8 @@ export type StoreMemoryResult = void;
 export type ApplyMemoryArguments = URI | undefined;
 export type ApplyMemoryResult = MemoryOptions;
 
+export type GetChildContextsResult = DebugRequestTypes['getChildContexts'][1];
```

Comment:
I'm still getting hung up on the word 'child' here. Maybe just `getContexts` or `getAvailableContexts`.